### PR TITLE
Adding Clickhouse Playground example

### PR DESCRIPTION
**Pull request by marlenezw**
This is an initial PR to solve [issue 2](https://github.com/ibis-project/ibis-examples/issues/2). This PR adds an example of connecting to a table through Clickhouse Playground and doing some basic analytics.

### CODE REVIEW

**Comment by marlenezw** [comment]
@jcrist here's the initial draft PR. Are there specific things you'd like me to show with the dataset? Happy to improve this in the morning

**Comment by danepitkin** [comment]
Nice! Shall we remove the `.DS_Store` files?

**Comment by jcrist** [comment]
yeah, I just added those to the `.gitignore` to prevent that happening in the future.

**Comment by marlenezw** [comment]
Just noticed that. I just used `git add .` I'll remove them!

File: examples/clickhouse_hackernews_example.ipynb [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+   ],
+   "source": [
+    "dish[['name', 'times_appeared']].order_by(ibis.desc('times_appeared'))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For more information on using Ibis with Clickhouse check out the docs page here: https://ibis-project.org/backends/ClickHouse/?h=click#clickhouse"
```
Comment (gforsyth) [inline examples/clickhouse_hackernews_example.ipynb:234]:
```suggestion
    "For more information on using Ibis with Clickhouse check out the docs page here: https://ibis-project.org/backends/ClickHouse/"
```

removing the highlighting of `Click` on the linked page

**Review by gforsyth — CHANGES_REQUESTED** [review]
Thanks for putting this together @marlenezw!

One small nit in a link, and those pesky `.DS_Store` files are still here.

Also, we don't currently have a policy, but I think we should probably run `black` over all the notebooks so that the code is formatted consistently between examples.

**Comment by jcrist** [comment]
>  but I think we should probably run black over all the notebooks so that the code is formatted consistently between examples.

Good idea Gil! I can push up a hook for the formatting changes with black. TIL black supports notebooks.

**Comment by jcrist** [comment]
Thanks Marlene! I've made a few changes to finish up this notebook. Merging!

**Comment by marlenezw** [comment]
Ah, sorry for the silence on this. Just catching up with Github notifications 😅  Thank you!!!

**Pull request merged**